### ISSUE TITLE
"Unable to extract file_key" on cloudy.ec

### ISSUE THREAD

**Issue by bmitch** [rg3/youtube-dl#5211]
Full verbose output:

```
youtube-dl http://www.cloudy.ec/v/7cd0da38be865 --verbose
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['http://www.cloudy.ec/v/7cd0da38be865', '--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.03.09
[debug] Python version 2.7.6 - Linux-3.13.0-45-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: none
[debug] Proxy map: {}
[Cloudy] 7cd0da38be865: Downloading webpage
ERROR: Unable to extract file_key; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 649, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/cloudy.py", line 108, in _real_extract
    r'filekey\s*=\s*"([^"]+)"', webpage, 'file_key')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 547, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract file_key; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by dstftw** [comment]
Will be fixed in the next version. Thanks for the report.

**Comment by dstftw** [comment]
Fixed in youtube-dl 2015.03.15. See [our FAQ](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) on how to update.
